### DISCUSSION TITLE
Prefix environment variables

### DISCUSSION THREAD

**Discussion by claytongulick** [directus/directus#15894] [Feature Requests]
### Describe the Bug

Directus uses dotenv amongst other mechanisms for configuration.

The environment variable names are generic, with names like:

```
HOST="0.0.0.0"
PORT=8055

DB_CLIENT="pg"
DB_HOST="localhost"
DB_PORT=5432
```

While convenient for containerized models, these have a high risk of collision in other deployment models.

A prefix like DIRECTUS_ would prevent collisions and namespace the environment variables.

To support both options, directus could look for both the prefixed and non-prefixed settings, preferring the prefixed one if it exists, falling back to the non-prefixed otherwise.

### To Reproduce

This is core system design.

### What version of Directus are you using?

9.18.1

### What version of Node.js are you using?

16.14.2

### What database are you using?

Postgres

### What browser are you using?

Chrome

### How are you deploying Directus?

locally

**Comment by br41nslug** [comment]
Moving this to a discussion to gauge community interest in this new feature/improvement 😄

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
